pipeline/scheduled-pipeline-executor.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+    Pipeline script for executing weekly scheduled test suites
+*/
+// Global variables section
+
+// Server to get recipe file contents
+def SSHServer = "ssh 10.245.4.89"
+
+// Recipe file path details
+def recipeFile = "RHCEPH-*.yaml"
+def recipeFileDir = "/data/site/recipe"
+
+// Default job parameters
+def buildType = "${params.buildType}" ?: "tier-1"
+def overrides = "${params.overrides}" ?: "{}"
+def tags = "${params.tags}" ?: ""
+def tagsList = tags.split(',') as List
+def nodeName = "centos-7"
+if ("ibmc" in tagsList) {
+    nodeName = "agent-01"
+}
+
+// Pipeline script entry point
+node(nodeName) {
+
+    stage("Trigger Pipeline") {
+        println "Begin synchronzing image artifacts."
+
+        def validRecipeFiles = sh(
+                returnStdout: true,
+                script: "${SSHServer} \"find ${recipeFileDir} -name ${recipeFile} -mtime -7 -print\"").trim().split("\n") as List
+
+        if (!validRecipeFiles) {
+            throw new RuntimeException("There no new RHCS build since last Friday")
+        }
+
+        for (validRecipeFile in validRecipeFiles) {
+            def rhcephVersion = validRecipeFile.split("/").last().replace(".yaml", "")
+            def recipeContent = sh(
+                    returnStdout: true,
+                    script: "${SSHServer} \"yq e '.latest' ${validRecipeFile}\"").trim()
```

Comment:
shouldn't it be `tier-0`? It helps to run against a smoke tested build.

Reply:
yes as `tier-0` records those that have passed Tier-0 test suites.

Reply:
Done